app/src/main/java/com/sdp13epfl2021/projmag/curriculumvitae/VideoCv.kt

```diff
@@ -0,0 +1,61 @@
+package com.sdp13epfl2021.projmag.curriculumvitae
+
+import android.app.Activity
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.net.Uri
+import android.os.Bundle
+import android.provider.MediaStore
+import android.widget.Button
+import android.widget.MediaController
+import android.widget.VideoView
+import androidx.appcompat.app.AppCompatActivity
+import com.sdp13epfl2021.projmag.R
+
+
+class VideoCv : AppCompatActivity() {
+
+    companion object {
+        const val RESULT_KEY = "com.sdp13epfl2021.projmag.curriculumvitea.VideoCV"
+    }
+    lateinit var vidButton: Button
+    lateinit var subButton: Button
+    lateinit var videoView: VideoView
+    private var VideoUri: Uri? = null
```

Comment:
variable should start with lower-case
```suggestion
    private var videoUri: Uri? = null
```